exports/scanresultscollector.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+import datetime
+import json
+import logging
+import os
+import random
+import string
+import tarfile
+import uuid
+from io import BytesIO
+from io import StringIO
+
+from google.cloud import storage
+
+
+class ScanResultsCollector:
+    """
+    Sends the result to a Google Cloud Storage bucket, for future use in DefectDojo
+    """
+
+    def __init__(self, keyfile, bucket_name, app_name, directory=None):
+        client = storage.Client.from_service_account_json(keyfile)
+        self.bucket = client.get_bucket(bucket_name)
+        self.directory = directory or f"RapiDAST-{app_name}"
+        self.app_name = app_name
+
+    def create_metadata(self, data):
+        """
+        Given a dictionary of key/values corresponding to Defectdojo's `import-scan` parameters,
+        return a Metadata dictionary
+        """
+        metadata = {
+            "scan_type": data["scan_type"],
+            "uuid": str(uuid.uuid1()),
+            "test_description": "This test was executed by RapiDAST",
```

Comment:
I'm wondering if this is necessary. If we want to identify the test as generated by RapiDAST, we could just add a tag to show it. My concern is that setting this description will require an additional DefectDojo API call, since the (re)import API can't set tests descriptions.

Reply:
We could NOT invoke another DD API call even if the description exists when we process it. However, if removing it would be making anything simpler, I agree with removing it as of now. And ideally, the description should be something that can be configured.

Reply:
removed